webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the character U+212B ANGSTROM SIGN will
```

Comment:
I think the example should be a capital A plus the ANGSTROM SIGN, as that's what would be normalized to Å.